test/integration/sign_up_test.rb

```diff
@@ -16,7 +16,7 @@ class SignUpTest < ActionDispatch::IntegrationTest
 
     post '/users', params: {
       'user' => { 'email' => 'test2@example.org', 'password' => 'frab12345', 'password_confirmation' => 'frab12345' },
-      'commit' => 'Sign up', 'conference_acronym' => @conference.acronym, 'locale' => 'en'
+      'commit' => 'Sign Up', 'conference_acronym' => @conference.acronym, 'locale' => 'en'
```

Comment:
This is probably a translation lookup problem (sign_up is not found and is capitalized).  it worked before refactoring, so you should not change the tests ;-)